README.md

```diff
@@ -87,11 +87,34 @@ Provide the configuration endpoint ([AWS Reference](http://docs.aws.amazon.com/A
 
 ```
 
-Multiple clusters Auto Discovery
-===============================
+If instead of using ElastiCache a custom auto discovery is needed you can provide your own function (specified as a MFA) that returns `{ok, list({Host :: inet:hostname(), Port :: inet:port_number()})}` to be used. Example:
+```erlang
+    [{cluster_b,
+        [{servers, {mfa, {myModule, myFunction, []}}},
```

Comment:
Can we replace `myModule` by `my_module` and `myFunction` by `my_function` here and everywhere else in this file? We generally prefer `snake_case` instead of `camelCase`.

Reply:
You're right. That's more "erlangish" ;)